doc/reference/tooling/tt_cli/cfg.rst

```diff
@@ -0,0 +1,35 @@
+Environment configuration
+=========================
+
+..  code-block:: bash
+
+    tt cfg [flags] <command> [command flags]
```

Comment:
1. Since there are no `cfg` command flags at this time, maybe remove `[flags]` for simplicity?
2. `<command>` should be `COMMAND` (upper case, no brackets) https://developers.google.com/style/code-syntax

We should generally align all CLI command syntax descriptions with Google or RedHat style guide, but it's a separate task.